<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelListView</class>
 <widget class="QWidget" name="ChannelListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ConfigObjectTableView" name="listView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="addAnalogChannel">
       <property name="text">
        <string>Add Analog Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDigitalChannel">
       <property name="text">
        <string>Add Digital Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cloneChannel">
       <property name="text">
        <string>Clone Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remChannel">
       <property name="text">
        <string>Delete Channel</string>
       </property>
       <property name="shortcut">
        <string>Alt+-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigObjectTableView</class>
   <extends>QWidget</extends>
   <header>configobjecttableview.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
